.github/workflows/rpm.yml

```diff
@@ -0,0 +1,33 @@
+name: RPM
+
+on:
+  push:
+    branches: [ marine ]
+  pull_request:
+    branches: [ marine ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: "Cache docker build"
+      uses: actions/cache@v2
+      id: cache-docker
+      with:
+        path: '/var/lib/docker/'
+        key: ${{ hashFiles('redhat/Dockerfile') }}-${{ hashFiles('redhat/rpms.txt') }}
+
+    - name: "Build RPMs"
+      run: cd redhat && bash ./redhat.sh
```

Comment:
I think `build.sh` is more appropriate than `redhat.sh` but not really important :)